client/src/pages/fundraisers/FundraiserCreate.js

```diff
@@ -169,35 +167,35 @@ class FundraiserCreate extends Component {
         fundraiser: {
           ...this.state.fundraiser,
           deadline: deadline_string,
-          image_url: this.state.imageUrl,
+          image_url: this.state.imageUrl
         }
       });
 
       // creating the updated fundraiser data
       const fundraiser_data = {
         ...this.state.fundraiser,
         deadline: deadline_string,
-        image_url: this.state.imageUrl,
-      }
+        image_url: this.state.imageUrl
+      };
 
       fetch("/fundraiser", {
         method: "POST",
         headers: {
           Accept: "application/json",
           "Content-Type": "application/json",
-          "Authorization": "Bearer " + token,
+          Authorization: "Bearer " + token
```

Comment:
If the linter removed the double quotes, ignore this comment -- otherwise put double quotes on both Authorization and Accept and that would be more consistent.

Reply:
Yes, this was a modification by the linter